### PR TITLE
cassandra: how to disable cross-cluster replication

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-166

This is a part of the PR set covering the cross-cluster replication feature for Aiven for Apache Cassandra.
By the time they are merged, I'm not able to cross-reference between them, which is obviously needed. When all the four articles are merged, I'll include relevant links in all of them so that the user is properly guided thru the whole thing.

- CCR concept https://github.com/aiven/devportal/pull/1759
- Enable CCR https://github.com/aiven/devportal/pull/1732
- Manage CCR https://github.com/aiven/devportal/pull/1741

### CODE REVIEW

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Split the services
+''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select the service for which you'd like to disable CCR.
+3. In the **Overview** tab of the service's page, navigate to **Cross Cluster Replication** and select **Split cluster**.
+
+   .. warning::
+
+      As soon as you split the cluster, the two services constituting the CCR pair become independent. It's not possible to recreate the CCR pair with the same service as a replica. You can select any other Aiven for Cassandra service for that purpose.
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.rst:47]: > You can select any other Aiven for Cassandra service for that purpose.

not quite that, you can:
* create 2 new services with CCR
* given an existing service, create a new one to do CCR with the existing service

but you cannot:
* given 2 existing services, enable CCR between them.

The quoted sentence sounds like "you cannot re-link the services that you just separated, but you can re-link one of them with another existing service" which is not the case

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+To disable CCR on your service, delete the replica service that is connected to your services for CCR purposes.
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of the service's page, navigate to the **Cross Cluster Replication** section and select the name of the replica service provided in the CCR description, which is supposed to take you to the replica service's page.
+4. In the **Overview** tab of the replica service, select **Delete service** from the top right corner.
+
+   .. warning::
+
+      As soon as you delete the replica service, CCR gets disabled for your service that is now connected to the replica service for CCR purposes.
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.rst:67]: > for your service that is now connected to the replica service for CCR purposes.

I think it's no longer connected to anything, after you deleted the other one?... I'd just end it at "CCR gets disabled" or maybe "your data is no longer replicated between regions"

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+Use the :ref:`avn service terminate <avn-cli-service-terminate>` command to disable CCR on your service by deleting its replica service.
+
+.. code-block:: bash
+
+   avn service terminate replica_service_name
+
+Disable CCR with API
+--------------------
+
+You can disable CCR for your Aiven for Apache Cassandra service(s) by calling the `terminate a service <https://api.aiven.io/doc/#tag/Service/operation/ServiceDelete>`_ endpoint to delete one of the services that constitute the CCR pair.
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.rst:93]: is it called "terminate endpoint" anywhere? it is ServiceDelete, maybe the "delete service endpoint"?

**Review by dmitry-potepalov — CHANGES_REQUESTED** [review]
overall ok, some concerns about the phrasing in the comments

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+.. important::
+
+   This private beta feature is available on request only. Contact sales@Aiven.io or support@Aiven.io to opt in. 
+
+This article provides you with instructions on how to discontinue the cross-cluster replication (CCR) for your Aiven for Apache Cassandra® service.
+
+About disabling CCR
+-------------------
+
+When you enable CCR for your service, you connect it to another service (replica service), which results in creating a CCR pair of services. You can disable CCR for your service either by splitting the services constituting the CCR pair or by deleting one of them.
```
[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/cassandra/howto/disable-cross-cluster-replication.rst:13]: if not replica service, how else can we name them? I mean the service on which you enable CCR and the other service you connect to for replication purposes? We could use some short terms for these guys. It's not easy for me to come up with relevant names: parent/primary/source/origin service vs child/secondary/sink/target service ?

[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/cassandra/howto/disable-cross-cluster-replication.rst]: @divdby0 , any ideas how to call the two service that build the CCR pair?

[the remark this answers is not in the document]
Comment (divdby0) [reply docs/products/cassandra/howto/disable-cross-cluster-replication.rst]: @wojcik-dorota In the Cassandra realm both parties are equals. I might suggest using _pairs_, _peers_ or just _services_. Although technically accurate, I agree with @dmitry-potepalov that using the term _replicas_ makes a fuzzy connotation to the PostgreSQL read-only replicas. Alternatively, we might get away with without using an explict term. i.e.: _When you enable CCR, your services get cross replicated with each other._

[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/cassandra/howto/disable-cross-cluster-replication.rst]: thx, will update accordingly :-)

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+Split the services
+''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select the service for which you'd like to disable CCR.
+3. In the **Overview** tab of the service's page, navigate to **Cross Cluster Replication** and select **Split cluster**.
+
+   .. warning::
+
+      As soon as you split the cluster, the two services constituting the CCR pair become independent. It's not possible to recreate the CCR pair connecting back to the same service. To enable CCR on your service again, you can create a new service and CCR-connect it to your existing service.
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.rst:43]: Should we include this warning in the About chapter as well?... on the one hand, it doesn't describe the splitting in detail, on the other hand - one might glance through the About chapter, nod and then proceed to click in the Console without ever scrolling down to see the warning :thinking:

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+* CLI
+* API
+
+It's recommended to use the Aiven console for disabling CCR.
+
+Prerequisites
+-------------
+
+* Aiven account
+* Depending on the method you choose to use for enabling CCR
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.rst:21]: "for enabling CCR" -> "for disabling CCR"

File: docs/products/cassandra/howto/disable-cross-cluster-replication.rst [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+   .. warning::
+
+      As soon as you delete the service where your data has been replicated, CCR gets disabled and your data is no longer replicated between regions.
+
+5. When in the **Delete confirmation** popup, make sure you understand the impact, copy-paste the service name, and select **Delete**.
+
+.. topic:: Result
+
+   You've disabled CCR on your service by deleting the service used as a sink for replicated data.
```
Comment (dmitry-potepalov) [inline docs/products/cassandra/howto/disable-cross-cluster-replication.rst:69]: > used as a sink for replicated data

hints that the other service is a source, meaning that the other service is somehow different; maybe "by deleting one of the peer services in a CCR service pair"?

**Review by dmitry-potepalov — COMMENTED** [review]
Overall LGTM, left some comments

**Review by alanfranz — APPROVED** [review]
LGTM

**Review by dmitry-potepalov — APPROVED** [review]
LGTM

**Review by divdby0 — APPROVED** [review]

**Pull request merged**